var/spack/repos/builtin/packages/mofem-cephas/package.py

```diff
@@ -24,15 +24,28 @@
 
 
 class MofemCephas(CMakePackage):
-    """mofem-cephas core library"""
+    """MoFEM is finite element core library"""
 
     homepage = "http://mofem.eng.gla.ac.uk"
-    git      = "https://bitbucket.org/likask/mofem-cephas.git"
+    url = "https://bitbucket.org/likask/mofem-cephas.git"
```

Comment:
Can you undo this change? This was likely due to a bad rebase. Spack now allows packages to specify a package-level `git` attribute so that you don't need to specify the same `git` for every version.